### PR TITLE
Multiple infiller decompose

### PR DESCRIPTION
**Pull request by Rlamboll**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [] Example added (either to an existing notebook or as a new notebook, where applicable) (not yet, next commit)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <https://github.com/znicholls/silicone/pull/XX>`_) Added feature which does something
- (`#XX <https://github.com/znicholls/silicone/pull/XX>`_) Fixed bug identified in (`#YY <https://github.com/znicholls/silicone/issues/YY>`_)
```

### CODE REVIEW

**Comment by Rlamboll** [comment]
This is to be done after #43

**Comment by Rlamboll** [comment]
The notebook where this is used is in PR #45 . It constructs a consistent basket in the infiller data and then breaks it down into several components.

File: src/silicone/multiple_infillers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Multiple infillers provide easy-to-use infiller options for the most common use-cases.
+"""
+
+from .decompose_collection_with_time_dep_ratio import (
+    DecomposeCollectionTimeDepRatio,
+)  # noqa: F401
+# from .infill_all_required_emissions_for_openscm import (
+#    infill_all_required_variables,
+#)  # noqa: F401
```
Comment (jkikstra) [inline src/silicone/multiple_infillers/__init__.py]: Do you want to keep this?

Comment (Rlamboll) [reply src/silicone/multiple_infillers/__init__.py]: It will get uncommented in the next PR

**Pull request merged**